pkg/volumereplica/volumereplica.go

```diff
@@ -385,6 +387,45 @@ func buildVolumeCloneCommand(cStorVolumeReplica *cstor.CStorVolumeReplica, snapN
 	return cloneVolCmd
 }
 
+func SetTargetIp(fullVolName, targetIp string) error {
+	stdoutStderr, err := RunnerVar.RunCombinedOutput(VolumeReplicaOperator,
+		"set", "io.openebs:targetip="+targetIp, fullVolName,
+	)
+
+	if err != nil {
+		klog.Errorf("Unable to set target ip for %s. error : %v", fullVolName, string(stdoutStderr))
+
+		return errors.Wrapf(err, "failed to set target ip: %s for %s", targetIp, fullVolName)
+	}
+
+	return nil
+}
+
+func GetTargetIp(fullVolName string) (string, error) {
+	stdoutStderr, err := RunnerVar.RunCombinedOutput(Execute, VolumeReplicaOperator,
```

Comment:
If we want to get/set any volume properties we can do it via [this](https://github.com/openebs/cstor-operators/blob/master/pkg/volumereplica/volumereplica.go#L669) using zcmd package.